### PR TITLE
[GE-2020] add speaker Mathieu Brun

### PR DESCRIPTION
**Pull request by deniger**
[GE-2020] add speaker Mathieu Brun

### CODE REVIEW

File: content/events/2020-geneva/speakers/mathieu-brun.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
++++
+Title = "Mathieu Brun"
+image = "mathieu-brun.jpg"
```
Comment (mattstratton) [inline content/events/2020-geneva/speakers/mathieu-brun.md]: FYI this image is not part of this PR, to the image is showing as a broken image in the deploy.

You should either add the image file, or remove this line to display the default image.

**Comment by mattstratton** [comment]
The image for this speaker is showing as a broken image, as it's not been added to this PR. 

image

**Comment by deniger** [comment]
thanks @mattstratton  for your help. I have just pushed a commit with the missing file

**Comment by mattstratton** [comment]
looks great! merging!
